contracts/validator/ValidatorSet.sol

```diff
@@ -126,6 +128,11 @@ contract ValidatorSet {
             validatorBeginHeight[_validator] == 0 && validatorEndHeight[_validator] == 0
```

Comment:
I still think that this should be `requires` and not `asserts`

Reply:
It was done because of the discussion at https://github.com/mosaicdao/mosaic-1/pull/212#discussion_r370237960